Library/Formula/sassc.rb

```diff
@@ -20,7 +20,7 @@ class Sassc < Formula
   depends_on "libsass"
 
   def install
-    ENV["SASSC_VERSION"] = "3.1.0"
+    ENV["SASSC_VERSION"] = "3.2.0"
```

Comment:
If this isn't set `sassc --version` would return no version string, since it's being extracted from a tarball (no git remote to query).